internal/providers/terraform/azure/cosmosdb_cassandra_keyspace.go

```diff
@@ -89,48 +85,63 @@ func provisionedCosmosCostComponents(model modelType, throughputs *decimal.Decim
 		meterName = "100 Multi-master RU/s"
 	}
 
+	name := "Provisioned throughput"
+	if model == Autoscale {
+		name = fmt.Sprintf("%s (autoscale", name)
+
+		if u != nil && u.Get("max_request_units_per_second").Exists() {
+			throughputs = decimalPtr(decimal.NewFromInt(u.Get("max_request_units_per_second").Int()))
+
+			if u.Get("max_request_units_utilization_percentage").Exists() {
```

Comment:
I like `max_request_units_utilization_percentage` that makes a lot of sense.  For `max_request_units_per_second` previously we were getting this value from `throughputs = decimalPtr(decimal.NewFromInt(d.Get("autoscale_settings.0.max_throughput").Int()))` what's the reason we can't still use that?

Reply:
Also:
If `max_request_units_utilization_percentage` is not set, we should show the 'Monthly cost depends on usage...'.  Right now if the precentage is not set it shows the max possible cost.  That doesn't seem correct.

Reply:
No reason, my bad. Brought it back and refactored it to show `'Monthly cost depends on ...'` if `max_request_units_utilization_percentage` is not set.

Reply:
@alikhajeh1 LGTM